sources/packages/backend/apps/db-migrations/src/migrations/1724457049888-DeleteFileContentAndMimeTypeColumns.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class DeleteFileContentAndMimeTypeColumns1724457049888
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Delete-file-content-and-mime-type-columns.sql",
+        "StudentFiles",
+      ),
+    );
+  }
+
+  public async down(): Promise<void> {
+    return;
```

Comment:
There was a misunderstanding with the statement "operation does not need a rollback planned.".
The point was that the data does not need to be rolled back because it would need an extra rollback operation.
The removed columns should be added back otherwise the upload/download will stop working.

Reply:
The `mime_type` and `file_content` are `NOT NULL` columns. Since we are not putting in any data back during rollback, should I make them `NULL` during rollback?

Reply:
The columns are originally not null and should be kept are they were.
They can be reintroduced with a default dummy value and have the default constant dropped.
mime_type can be defined as "text/plain" and "file_content" as "Deleted".

Reply:
Apologies, I missed to update it after our call. Updated now.